src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -43,13 +47,29 @@ public class PartialTagHelper : TagHelper
         /// An expression to be evaluated against the current model. Cannot be used together with <see cref="Model"/>.
         /// </summary>
         [HtmlAttributeName(ForAttributeName)]
```

Comment:
Historical curiosity: This name matches the default. Why are we explicit here?

Reply:
The constant's used in an error message so I maybe this was left untouched to support it ❓